Create ui-c8y-1023-82-3-restore-angular-js-event-list-implementation-12069-graft-release-cd-12083.md (#4748)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-82-3-restore-angular-js-event-list-implementation-12069-graft-release-cd-12083.md

---
date: ""

> title: Restored AngularJS Event list implementation

Reviewer: ```suggestion
title: Restored AngularJS event list
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66883
version: 1023.82.3
---

> Restored the AngularJS event list in @c8y/ng1-modules, removed during the migration to the new Event list. Its removal broke custom applications scaffolded before the migration that still reference the old module in their own bootstrap, causing build failures after a Web SDK upgrade.

Reviewer: ```suggestion
The AngularJS event list in @c8y/ng1-modules has been restored, which was removed during the migration to the new event list. Its removal broke custom applications scaffolded before the migration that still reference the old module in their own bootstrap, causing build failures after a Web SDK upgrade.
```

> The AngularJS event list in @c8y/ng1-modules has been restored, which was removed during the migration to the new event list. Its removal broke custom applications scaffolded before the migration that still reference the old module in their own bootstrap, causing build failures after a Web SDK upgrade.

Reviewer: @luja-c8y Does "new event list" mean "Angular event list"?

Reviewer 2: @BeateRixen yes

> The AngularJS event list in @c8y/ng1-modules has been restored, which was removed during the migration to the new event list. Its removal broke custom applications scaffolded before the migration that still reference the old module in their own bootstrap, causing build failures after a Web SDK upgrade.

Reviewer: ```suggestion
The former AngularJS event list in @c8y/ng1-modules has been restored. It was removed during the migration to the new Angular event list. Its removal broke custom applications scaffolded before the migration that still reference the old module in their own bootstrap, causing build failures after a Web SDK upgrade.
```

---
date: ""

> title: restore AngularJS event list implementation (#12069) [GRAFT][release/cd] (#12083)

Reviewer 2: ```suggestion
title: Restored AngularJS Event list implementation
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66883
version: 1023.82.3
---
# Backport

This will backport the following commits from `develop` to `release/cd`:
- [fix(Web SDK): [MTM-66883] restore AngularJS event list implementation
(#12069)](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/12069)

<!--- Backport version: 9.5.1 -->

### Questions ?
Please refer to the [Backport tool
documentation](https://github.com/sorenlouv/backport)

[MTM-66883]:
https://cumulocity.atlassian.net/browse/MTM-66883?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Co-authored-by: Lukasz Janusz <114157358+luja-c8y@users.noreply.github.com>

> Co-authored-by: Lukasz Janusz <lukasz.janusz@cumuloicty.com>

Reviewer 2: ```suggestion
Restored the AngularJS event list in @c8y/ng1-modules, removed during the migration to the new Event list. Its removal broke custom applications scaffolded before the migration that still reference the old module in their own bootstrap, causing build failures after a Web SDK upgrade.
```
